setup.py

```diff
@@ -76,7 +76,7 @@ def get_git_sha():
         'contextlib2',
         'elasticsearch',
         'croniter==0.3.25',
-        'cryptography>=2.4.2',
+        'cryptography<=2.4.2',
```

Comment:
why we have put `<` instead?

Reply:
so that it gets aligned with version mentioned in  Pipfile

Reply:
As discussed, please revert to original, i.e. `cryptography>=2.4.2`